src/main/java/com/ays/backend/user/controller/payload/request/SignUpRequest.java

```diff
@@ -0,0 +1,41 @@
+package com.ays.backend.user.controller.payload.request;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+/**
+ * Signup request coming to the API layer
+ */
+@Data
+@Builder
+@AllArgsConstructor
+public class SignUpRequest {
+
+    @NotBlank
+    private String username;
+
+    @NotBlank
+    private String password;
+
+    @NotNull
+    private int countryCode;
+
+    @NotNull
+    private int lineNumber;
+
+    @NotNull
+    private int statusId;
+
+    @NotBlank
+    private String firstName;
+
+    @NotBlank
+    private String lastName;
+
+    private String email;
+
+    private Long organizationId;
```

Comment:
Do we want to request this through signup or in fact through any user-facing end point? We removed role so a user couldn't set themselves as admin or worker. Now that the only option is `ROLE_VOLUNTEER`, do we want to take this here? I suppose we can let volunteers be a part of organizations they choose, but that would have to be an array, and we might want to consider moving the field to another flow after signup.

Reply:
I mean `organizationId`.